types/trigger.d.ts

```diff
@@ -35,12 +35,21 @@ export type ZoneIdType = number | null;
 
 export type OutputStrings = { [outputKey: string]: LocaleText | string };
 
+export type OutputStringsParamObject = {
+  [key: string]: string | number | (() => string);
```

Comment:
Mmmm is there support for `() => string` in the code?

Reply:
Oh, that makes sense.  Can you leave a comment that `() => string` is only valid for `toString`?